classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_student(student)
+    @student.push(student)
+    stude.classroom = self
```

Comment:
- [ ] I believe this is a typo, kindly consider fixing this `student.classrom = self` so your method is functioning correctly